newswires/app/db/FingerpostWireEntry.scala

```diff
@@ -275,23 +276,26 @@ object FingerpostWireEntry
       case _ => None
     }
 
-    // grr annoying but broadly I think subjects(/categoryCodes) and keywords are the same "axis" to search on
```

Comment:
Removed because keywords can be combined with category codes.
Ex: Soccer `Category: apCat:s` + `Keyword: Soccer`